src/item.cpp

```diff
@@ -6506,13 +6512,17 @@ bool item::can_contain( const item &it ) const
         // does the set of all sets contain itself?
         return false;
     }
-    for( const item *internal_it : contents.all_items_top( item_pocket::pocket_type::CONTAINER ) ) {
-        if( internal_it->contents.can_contain_rigid( it ).success() ) {
+
+    for( const item *internal_it : contents.all_items_top(item_pocket::pocket_type::CONTAINER) ) {
+        if(internal_it->contents.can_contain_rigid(it).success()) {
             return true;
         }
     }
+    if (is_fuel_for(it)) {
```

Comment:
can't we just adjust the json to contain things properly, instead of writing code to re-add the old charge behavior?